tests/app/main/test_briefs.py

```diff
@@ -120,6 +129,7 @@ def test_clarification_question_checks_supplier_is_eligible(self, data_api_clien
         self.login()
         data_api_client.get_brief.return_value = api_stubs.brief(status='live')
         data_api_client.get_brief.return_value['briefs']['frameworkName'] = 'Digital Outcomes and Specialists'
+        data_api_client.get_brief.return_value['briefs']['lot'] = 'digital-specialists'
```

Comment:
`api_stubs.brief` has `digital-specialists` as the lot by default so there's no need to set it here.

To make it explicit for the test (which I agree is nice) you can pass it in to the stubs constructor, eg: `api_stubs.brief(status='live', lot_slug='digital-specialists')`